resources/deletedb.yaml

```diff
@@ -1,3 +1,45 @@
+---
```

Comment:
I was aiming for long-term separation of concern so that we can eventually narrow them down further (if needed/wanted) without affecting all the pods using them.